### PR TITLE
Set sockd to executable

### PR DESCRIPTION
**Pull request by thomasm-ttd**

### CODE REVIEW

File: .github/workflows/publish-aws-eks-nitro-enclave-docker.yaml [hunk 1]
```diff
@@ -89,15 +91,17 @@ jobs:
...
-          echo ${{ needs.buildEUIDImage.outputs.enclave_id }} >> ${{ env.ARTIFACTS_BASE_OUTPUT_DIR }}/manifests/aws-eks-euid-enclave-id-${{ needs.buildEUIDImage.outputs.eif_version_number }}.txt
+          echo "EUID EIF Version: ${{ needs.buildEUIDImage.outputs.eif_version_number }}" > ${{ env.ARTIFACTS_BASE_OUTPUT_DIR }}/manifests/aws-eks-euid-enclave-id-${{ needs.buildEUIDImage.outputs.image_tag }}.txt
+          echo "EUID Image Tag: ${{ needs.buildEUIDImage.outputs.image_tag }}" >> ${{ env.ARTIFACTS_BASE_OUTPUT_DIR }}/manifests/aws-eks-euid-enclave-id-${{ needs.buildEUIDImage.outputs.image_tag }}.txt
+          echo "Enclave ID (maybe shared by other images): " ${{ needs.buildEUIDImage.outputs.enclave_id }} >> ${{ env.ARTIFACTS_BASE_OUTPUT_DIR }}/manifests/aws-eks-euid-enclave-id-${{ needs.buildEUIDImage.outputs.image_tag }}.txt
 
       - name: Save Manifests as build artifacts
         uses: actions/upload-artifact@v4
         with:
-          name: aws-eks-enclave-ids-${{ needs.buildUID2Image.outputs.eif_version_number }}
+          name: aws-eks-enclave-ids-${{ needs.buildUID2Image.outputs.image_tag }}
```
Comment (cYKatherine) [inline .github/workflows/publish-aws-eks-nitro-enclave-docker.yaml:105]: Does `buildUID2Image.outputs.image_tag` guarenteed to be the same as `buildEUIDImage`?

Comment (thomasm-ttd) [reply .github/workflows/publish-aws-eks-nitro-enclave-docker.yaml:105]: Yes, as they come from the same source.

File: scripts/aws/eks-pod/sockd_eks.conf [hunk 2]
```diff
@@ -1,3 +1,6 @@
+#logoutput: stdout
```
Comment (cYKatherine) [inline scripts/aws/eks-pod/sockd_eks.conf:1]: What does this block do?

Comment (thomasm-ttd) [reply scripts/aws/eks-pod/sockd_eks.conf:1]: currently, only logs errors to the stdout, but has comments to easily enable debug output

**Review by cYKatherine — APPROVED** [review]

**Pull request merged**